src/transformers/models/big_bird/modeling_flax_big_bird.py

```diff
@@ -966,9 +982,12 @@ def _bigbird_block_rand_mask(
 
         if from_seq_length // from_block_size != to_seq_length // to_block_size:
             raise ValueError("Error the number of blocks needs to be same!")
+        rand_attn = jnp.zeros((from_seq_length // from_block_size - 2, num_rand_blocks), dtype=jnp.int32)
+        # deterministic nor randomness
+        if indices_prng_key is None:
```

Comment:
I guess this `if` statement could also be:
```python
if deterministic:
```
Since `indices_prng_key` is None when we're deterministic. Happy with either! Think the `if deterministic` might be easier to understand logic wise?